internal/cmdrender/runtime/container.go

```diff
@@ -24,15 +24,18 @@ import (
 
 	"github.com/GoogleContainerTools/kpt/internal/printer"
 	"github.com/GoogleContainerTools/kpt/internal/types"
+	"github.com/GoogleContainerTools/kpt/internal/util/fnoutput"
 )
 
 // containerNetworkName is a type for network name used in container
 type containerNetworkName string
 
 const (
-	networkNameNone containerNetworkName = "none"
-	networkNameHost containerNetworkName = "host"
-	defaultTimeout  time.Duration        = 5 * time.Minute
+	networkNameNone     containerNetworkName = "none"
+	networkNameHost     containerNetworkName = "host"
+	defaultTimeout      time.Duration        = 5 * time.Minute
+	fnIdentation                             = 2
```

Comment:
This should be in the printer. At some point we might want to use different implementations of the printer interface (like providing a printer that uses ascii color codes), so I think we should try to move all functionality related to output into the printer. That probably means making the printer api a little more specific to functions/packaging/live.

Reply:
+1.
Also we can make this flag `fn` agnostic.  `printer.Indent` ?

Reply:
What I thought was different command contents may have different indentations.

Reply:
Moved these constants to `printer`.